locale/fr_FR/LC_MESSAGES/pdfstitcher.po

```diff
@@ -253,15 +253,14 @@ msgstr "Remise a zéro"
 
 #: pdfstitcher.py:303
 msgid "Apply to checked"
-msgstr ""
-
+msgstr "Appliquez a la selection
```

Comment:
I think the closing `"` is missing here

Reply:
I'm not a native speaker, but isn't it rather `à` than `a` in this case?

Reply:
Sometimes quotes get dropped, I'll test it out tonight. Since translators aren't usually programmers I'll go through and make sure everything builds correctly without taking more of their time :)

Reply:
Okay, fine.